app/services/logger.js

```diff
@@ -0,0 +1,43 @@
+const bunyan = require('bunyan');
+const PrettyStream = require('bunyan-prettystream');
+const { get } = require('lodash');
+
+const Logger = require('./base_logger');
+
+const pkg = require('../../package.json');
```

Comment:
Why we need to import `package.json`? Is it correct to chose between config.name and pkg.name?

Reply:
wow-wow, package.json!?

Reply:
wow-wow, package.json!?

Reply:
[Fixed](https://github.com/inst-event-emitter/eventemitter-backend/pull/7/commits/44502c79cf19007b92f9f6739774ab39f2674789#diff-2f876aef44739dc3746ddb1485ae379fR12)